katago_server/trainings/models/network.py

```diff
@@ -60,6 +91,7 @@ class Meta:
     )  # used to select best unsure network for ranking games (matches)
 
     def __str__(self):
+        # QUESTION (lightvector): self.id is a mismatch with uuid? Is there also an "id" field too?
```

Comment:
There is also an id, which is by default a primary key, integer autoincrementing.